src/stories/containers/TransparencyReport/components/BudgetReport/BudgetReport.tsx

```diff
@@ -0,0 +1,144 @@
+  const [actualsBreakdownSelected, setActualsBreakdownSelected] = useState<string | undefined>();
+
+  const handleActualsBreakdownChange = (current?: string) => {
+    setActualsBreakdownSelected(current);
+  };
+
+  return (
+    <BudgetReportWrapper>
+      <ActualsSection>
+        <Container>
+          <LinkDescription isLight={isLight}>
+            <span> Visit makerburn.com to</span>
+            <ActualViewOnChainLink
+              href={`${MAKER_BURN_LINK}/${longCode}`}
+              fontSize={16}
+              fontWeight={500}
+              iconWidth={10}
+              iconHeight={10}
+              marginLeft="7px"
+            >
+              {`view the ${code} Core Unit on-chain transaction history`}
+            </ActualViewOnChainLink>
+
+            <BudgetDateTitle isLight={isLight}>{currentMonth.toFormat('MMMM yyyy')} Budget Report</BudgetDateTitle>
+          </LinkDescription>
+        </Container>
+
+        <BudgetSection title={'Actuals - Totals'}>
+          <AdvancedInnerTable
+            columns={actualsData.mainTableColumns}
+            items={actualsData.mainTableItems}
+            cardsTotalPosition="top"
+            longCode={longCode}
+          />
+
+          {actualsData.mainTableItems.length > 0 && (
+            <>
+              <TitleSpacer>
+                <SectionTitle>Actuals Breakdown</SectionTitle>
+              </TitleSpacer>
+
+              <Tabs
+                tabs={actualsData.breakdownTabs.map((header, i) => ({
+                  item: header,
+                  id: actualsData.headerIds[i],
+                }))}
+                expandable
+                expandedDefault={false}
+                viewKey={'breakdownView'}
+                onChange={handleActualsBreakdownChange}
+              />
+
+              {actualsBreakdownSelected ? (
+                <AdvancedInnerTable
+                  columns={actualsData.breakdownColumns}
+                  items={actualsData.breakdownItems}
+                  longCode={longCode}
+                  style={{ marginTop: 16 }}
+                  tablePlaceholder={<TransparencyEmptyTable breakdown longCode={longCode} />}
+                />
+              ) : (
+                <BudgetSection level={2}>
+                  {actualsData.breakdownTabs.map((header) => (
+                    <div key={header}>
+                      <SectionTitle level={2} hasIcon={true}>
+                        {header}
+                      </SectionTitle>
+                      <div>table here...</div>
+                    </div>
+                  ))}
+                </BudgetSection>
+              )}
+            </>
+          )}
+        </BudgetSection>
+      </ActualsSection>
+    </BudgetReportWrapper>
+  );
+};
+
+export default BudgetReport;
+
+const BudgetReportWrapper = styled.div({});
+
+const ActualsSection = styled.div({});
+
+const ActualViewOnChainLink = styled(CustomLink)({
+  flexWrap: 'wrap',
```

Comment:
Does this flex-wrap: 'wrap',  have some effect on this line

Reply:
No, this has no effect on this component as it is displayed as `inline-block`. Deleted the line